modules/efk-logging-configuring-image-about.adoc

```diff
@@ -18,11 +18,23 @@ oc -n openshift-logging set env deployment/cluster-logging-operator --list | gre
 You can overwrite the image for each cluster logging component by modifying the appropriate
 environment variable in the `cluster-logging-operator` CR. 
 
-* *ELASTICSEARCH_IMAGE* deploys Elasticsearch, by default `quay.io/openshift/origin-logging-elasticsearch5:latest`.
-* *FLUENTD_IMAGE* deploys Fluentd, by default `quay.io/openshift/origin-logging-fluentd:latest`.
-* *KIBANA_IMAGE* deploys Kibana, by default `quay.io/openshift/origin-logging-kibana5:latest`.
-* *CURATOR_IMAGE* deploys Curator, by default `quay.io/openshift/origin-logging-curator5:latest`.
-* *OAUTH_PROXY_IMAGE* defines OAUTH for {product-title}, by default `quay.io/openshift/origin-oauth-proxy:latest`.
+----
+oc -n kube-system set env deployment/cluster-logging-operator --list | grep _IMAGE
```

Comment:
The cluster-logging-operator should be deployed to namespace `openshift-logging`.